src/MetaTags/Description.php

```diff
@@ -56,11 +56,16 @@ public function get_singular_value( $post_id = null ) {
 		$data = get_post_meta( $post_id, 'slim_seo', true );
 		if ( ! empty( $data['description'] ) ) {
 			$this->is_manual = true;
+			
+			/** This mimics the behaviour of `get_the_excerpt` in `/wp-includes/post-template.php` */
+			if ( post_password_required( $post_id ) ) {
+				return __( 'There is no excerpt because this is a protected post.' );
```

Comment:
Can you please add text domain to this line?

Reply:
Adding the 'slim-seo' text-domain would create a different translation-string than the one in [`get_the_excerpt`](https://github.com/WordPress/WordPress/blob/master/wp-includes/post-template.php#L415) which uses the core translation.

If you want to add both occurences of the translation inside the text-domain of slim-seo we could use the following code instead of calling `get_the_excerpt` itself:

```
$post = get_post( $post_id );
if ( empty( $post ) ) {
	return '';
}

if ( post_password_required( $post ) ) {
	return __( 'There is no excerpt because this is a protected post.', 'slim-seo' );
}

// Apply the WordPress excerpt filter
return apply_filters( 'get_the_excerpt', $post->post_excerpt, $post );
```
Let me know what you think. I'll update the branch accordingly.

Reply:
I moved the password protection check to the top of the function in my latest commit. This would be the best solution I think :)